.github/workflows/pvs-studio.yml

```diff
@@ -35,6 +35,12 @@ jobs:
           sudo dpkg -i "pvs-package/pvs.deb"
           pvs-studio-analyzer credentials "${{ secrets.PvsStudioName }}" "${{ secrets.PvsStudioKey }}"
 
+      - name: Install Meson via pip3
+#        if:   matrix.conf.os != 'ubuntu-20.04'
```

Comment:
(minor) might as well knock this out instead of commenting it (and the commit comment will explain why)